test/functional/fixtures/api/es-next/click/test.js

```diff
@@ -4,10 +4,10 @@ var expect = require('chai').expect;
 // Actions functionality is tested in lower-level raw API.
 describe('[API] t.click()', function () {
     it('Should make click on a button [ONLY:chrome]', function () {
-        return runTests('./testcafe-fixtures/click-test.js', 'Click button', { shouldFail: true })
+        return runTests('./testcafe-fixtures/click-test.js', 'Click button', { shouldFail: true, only: 'chrome' })
             .catch(function (err) {
-                expect(err).to.contains('Button clicked');
-                expect(err).to.contains(
+                expect(err[0]).to.contains('Button clicked');
+                expect(err[0]).to.contains(
```

Comment:
Here and in all other cases: rename `err` var to `errs` since it's now always contains multiple errors